### PR TITLE
[DBC-213] Wallet list dialog

### PR DESCRIPTION
**Pull request by quocnd**
## Description
- Implement UI for wallet list dialog

## Screenshots

image

### CODE REVIEW

File: app/src/main/java/io/mvlchain/tada/wallet/app/ui/wallet/WalletListDialog.kt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import androidx.core.os.bundleOf
+import androidx.core.view.isVisible
+import androidx.fragment.app.setFragmentResult
+import io.mvlchain.tada.wallet.app.base.BaseBottomSheetDialogFragment
+import io.mvlchain.tada.wallet.app.base.BindingInflater
+import io.mvlchain.tada.wallet.app.di.GlideApp
+import io.mvlchain.tada.wallet.databinding.DialogWalletListBinding
+import io.mvlchain.tada.wallet.model.WalletModel
+
+class WalletListDialog : BaseBottomSheetDialogFragment<DialogWalletListBinding>() {
```
Comment (cutehackers) [inline app/src/main/java/io/mvlchain/tada/wallet/app/ui/wallet/WalletListDialog.kt:27]: Rename this screen from `WalletListDialog` to `WalletListDialogFragment`

**Review by cutehackers — CHANGES_REQUESTED** [review]
Minor changes required.

**Comment by cutehackers** [comment]
I just wonder if the height of the WalletList can be dynamically changed?

File: app/src/main/java/io/mvlchain/tada/wallet/app/ui/wallet/WalletListDialog.kt (hunk 1 above)
Comment (quocnd) [reply app/src/main/java/io/mvlchain/tada/wallet/app/ui/wallet/WalletListDialog.kt:27]: Correct one, will change it

**Comment by quocnd** [comment]
> I just wonder if the height of the WalletList can be dynamically changed?

It can be but we need to handle programmatically by Kotlin. It can't be arranged easily on layout. So I think we can improve this later
E.g:
- We can't use `layout_height="0"` because recyclerView is not visible with list
- If we use `layout_height="wrap_content"`: If the list exceeds screen height, the list will overlap other views.
image

**Review by cutehackers — APPROVED** [review]
Okay great, Please leave a comment for the height of the wallet list recyclerview.

**Pull request merged**